C/services/filter-plugin-interfaces/python/python_plugin_interface.cpp

```diff
@@ -161,32 +170,44 @@ static void filter_plugin_reconfigure_fn(PLUGIN_HANDLE handle,
  */
 void filter_plugin_ingest_fn(PLUGIN_HANDLE handle, READINGSET *data)
 {
-	if (!pythonModules)
+        if (!handle)
 	{
-		Logger::getLogger()->error("pythonModules map is NULL "
-					   "in filter_plugin_ingest_fn plugin '%s'",
-					   gPluginName.c_str());
+		Logger::getLogger()->fatal("plugin_handle: filter_plugin_ingest_fn(): "
+					   "handle is NULL");
 		return;
 	}
 
-	auto it = pythonModules->find(gPluginName);
-	if (it == pythonModules->end() ||
+	if (!pythonHandles)
+	{
+		// Plugin name can not be logged here
+		Logger::getLogger()->error("pythonHandles map is NULL "
+					   "in filter_plugin_ingest_fn");
+		return;
+	}
+
+	auto it = pythonHandles->find(handle);
+	if (it == pythonHandles->end() ||
 	    !it->second)
 	{
+		// Plugin name can not be logged here
 		Logger::getLogger()->fatal("plugin_handle: plugin_ingest(): "
-					   "pModule is NULL for plugin '%s'",
-					   gPluginName.c_str());
+					   "pModule is NULL");
```

Comment:
We seem to have reduce the information here, is the deliberate?

Reply:
Plugin name cannot be logged if pythonHandles map is not available or handle does not exist un the map